AddUtil.java

```diff
@@ -14,13 +14,67 @@ public static void myassert(boolean  x) {
             throw new IllegalArgumentException("Assert fail") ;
         }
     }
-    private static int add(int a, int b){
+    private static int add(int a, int b)
+    {
         //Write your code here
+        System.out.println(a+b);
+        return a + b;
     }
 
+
     private static String add(String a, String b) {
         //Write your code here
+        Stack myStack = new Stack();
+        int carry = 0;
+        int len1 = String.valueOf(a).length();
+        int len2 = String.valueOf(b).length();
+        len1 = len1 - 1;
+        len2 = len2 - 1;
+        int s = 0, sum;
+        int e,f;
+        char c,d;
+        String result = "";
+
+        while(true){
```

Comment:
While(true) - Not a recommended approach
``
Always to try to put a condition inside the while loop.
In this case it would be ((len1) < 0 && (len2) < 0) from line 62